core-ui/src/components/Predefined/Details/Pod/Pod.details.js

```diff
@@ -126,6 +126,7 @@ export const PodsDetails = ({ DefaultRenderer, ...otherParams }) => {
         Events,
       ]}
       customColumns={customColumns}
+      grafanaQuery={`{pod="${otherParams.resourceName}"}`}
```

Comment:
we should also add a namespace to the query, you can have pod with the same name in multiple namespaces